test/parallel/test-dns.js

```diff
@@ -245,3 +245,24 @@ assert.throws(() => {
   code: 'ERR_INVALID_CALLBACK',
   type: TypeError
 }));
+
+{
+  // Fix #14734
```

Comment:
Another nit: When referring to issues/PRs, please use the full URL when possible :)